cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -62,6 +66,7 @@ class RandomAccessObject : public arrow::io::RandomAccessFile {
 
   arrow::Result<int64_t> ReadAt(int64_t position, int64_t nbytes, void* out) { return 0; }
 
+  /// Read the specified position. Return a buffer of the objects at the position.
```

Comment:
We can just say, "Read at a specified number of bytes from a specified position"

Reply:
Done.